projects/aca-content/src/lib/ui/theme.scss

```diff
@@ -1,3 +1,4 @@
+/* stylelint-disable selector-class-pattern */
```

Comment:
Do you guys think it's allowable to disable that styleline rule for theme files? I think it's really difficult to modify those files as they mainly focus on adapting mat classes

Reply:
I think it's okay for me